########## Testcase 0: Failed ##########
Expected Output: 
0

Your Output: 
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 5
	at Solution.main(Solution.java:36)
----------------------------------------

########## Testcase 1: Failed ##########
Expected Output: 
3

Your Output: 
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 5
	at Solution.main(Solution.java:36)
----------------------------------------

########## Testcase 2: Failed ##########
Expected Output: 
7

Your Output: 
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 11
	at Solution.main(Solution.java:36)
----------------------------------------

